project.php

```diff
@@ -153,6 +153,7 @@ class="btn-large grey">
 
 		</div>
 	</div>
+	</div>
```

Comment:
do you really want this?  I might delete it.

Reply:
hmmm... I thought it would fix something to remove it, but it didn't.  Did you have a reason for adding this?

Reply:
ha, removing it did fix it but it was caching.  Maybe I can put in some sort of debug flag on that cache command...

Reply:
which button was this? i'm confused.